src/main/java/net/dv8tion/jda/api/requests/ErrorResponse.java

```diff
@@ -37,83 +37,148 @@
+    INVITE_CODE_INVALID(                      50020, "Invite code is either invalid or taken"),
+    INVALID_MESSAGE_TARGET(                   50021, "Cannot execute action on a system message"),
+    INVALID_CHANNEL_TYPE(                     50024, "Cannot execute action on this channel type"),
+    INVALID_OAUTH_ACCESS_TOKEN(               50025, "Invalid OAuth2 access token"),
+    MISSING_OAUTH_SCOPE(                      50026, "Missing required OAuth2 scope"),
+    INVALID_WEBHOOK_TOKEN(                    50027, "Invalid Webhook Token"),
+    INVALID_ROLE(                             50028, "Invalid role"),
+    INVALID_RECIPIENT(                        50033, "Invalid Recipient(s)"),
+    INVALID_BULK_DELETE_MESSAGE_AGE(          50034, "A Message provided to bulk_delete was older than 2 weeks"),
+    INVALID_FORM_BODY(                        50035, "Invalid Form Body"),
+    INVITE_FOR_UNKNOWN_GUILD(                 50036, "An invite was accepted to a guild the application's bot is not in"),
+    INVALID_API_VERSION(                      50041, "Invalid API version"),
+    FILE_UPLOAD_MAX_SIZE_EXCEEDED(            50045, "File uploaded exceeds the maximum size"),
+    INVALID_FILE_UPLOADED(                    50046, "Invalid file uploaded"),
+    CANNOT_SELF_REDEEM_GIFT(                  50054, "Cannot self-redeem this gift"),
+    PAYMENT_SOURCE_REQUIRED(                  50070, "Payment source required to redeem gift"),
+    CANNOT_DELETE_CHANNEL_COMMUNITY(          50074, "Cannot delete a channel required for Community guilds"),
+    INVALID_STICKER_SENT(                     50081, "Invalid Sticker Sent"),
+    ILLEGAL_OPERATION_ARCHIVED_THREAD(        50083, "Tried to perform an operation on an archived thread, such as editing a message or adding a user to the thread"),
+    INVALID_THREAD_NOTIFICATION_SETTINGS(     50084, "Invalid thread notification settings"),
+    BEFORE_VALUE_EARLIER_THAN_THREAD_CREATION(50085, "\"before\" value is earlier than the thread creation date"),
+    SERVER_NOT_AVAILABLE_IN_YOUR_LOCATION(    50095, "This server is not available in your location"),
+    SERVER_MONETIZATION_DISABLED(             50097, "This server needs monetization enabled in order to perform this action"),
+    SERVER_NOT_ENOUGH_BOOSTS(                 50101, "This server needs more boosts to perform this action"),
+    MFA_NOT_ENABLED(                          60003, "MFA auth required but not enabled"),
+    NO_USER_WITH_TAG_EXISTS(                  80004, "No users with DiscordTag exist"),
+    REACTION_BLOCKED(                         90001, "Reaction Blocked"),
+    RESOURCES_OVERLOADED(                    130000, "Resource overloaded"),
+    STAGE_ALREADY_OPEN(                      150006, "The Stage is already open"),
+    MISSING_READ_MESSAGE_HISTORY_PERM(       160002, "Cannot reply without permission to read message history"),
```

Comment:
This name seems to be in conflict with its description. This naming could result it in it implying that it applies to many things.